xcpretty.gemspec

```diff
@@ -26,6 +26,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
+  spec.add_dependency 'rouge', '~> 1.8'
+
```

Comment:
@supermarin @kylef In any case, coloring is disabled but xcp still works if rouge is not installed, so I think we are good here?